force-app/main/default/lwc/propertyLocation/__tests__/data/getRecord.json

```diff
@@ -2,7 +2,7 @@
     "apiName": "Property__c",
     "childRelationships": {},
     "id": "a015I000002taYUQAY",
-    "lastModifiedById": "0055I000001RjtIQAS",
+    "lastModifiedById": "0054I000001RjtIQAS",
```

Comment:
@albarivas just double-checking, some of these are `0055`... assuming it doesn't matter much but in case wanted to know if these should be rechanged to `0055` in the cases where it's `55` instead of `54`?

Reply:
It doesn't matter, it's just a fake Id, I used one different from 55 so that next release cycle we don't have the same problem!